backend/dataall/modules/dataset_sharing/services/data_sharing_service.py

```diff
@@ -217,4 +268,17 @@ def revoke_share(cls, engine: Engine, share_uri: str):
                 new_share_state = share_sm.run_transition(ShareObjectActions.Finish.value)
             share_sm.update_state(session, share, new_share_state)
 
-            return revoked_tables_succeed and revoked_folders_succeed
+            return revoked_folders_succeed and revoked_s3_buckets_succeed and revoked_tables_succeed
+
+    @staticmethod
+    def _handle_table_share_failure(session, share, table, share_item_status):
```

Comment:
Is this method called anywhere `_handle_table_share_failure()`

Reply:
Nope removing. This will be added as a part of Central catalog support PR.